addOns/httpsInfo/src/main/java/org/zaproxy/zap/extension/httpsinfo/UsagePanel.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2020 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.httpsinfo;
+
+import java.awt.*;
+import javax.swing.*;
+import org.parosproxy.paros.extension.AbstractPanel;
+import org.parosproxy.paros.extension.ViewDelegate;
+import org.parosproxy.paros.model.Model;
+import org.parosproxy.paros.view.View;
+
+public class UsagePanel extends AbstractPanel {
+
+    private static final long serialVersionUID = 2L;
+    private final ViewDelegate view;
+    private JPanel panel;
+    private JTextArea tf;
+
+    public UsagePanel(View view) {
+        super();
+        this.view = view;
+        initialize();
+    }
+
+    private void initialize() {
+        this.setLayout(new BorderLayout());
+        if (Model.getSingleton().getOptionsParam().getViewParam().getWmUiHandlingOption() == 0) {
+            this.setSize(600, 200);
+        }
+        this.add(getHistoryPanel(), java.awt.BorderLayout.CENTER);
+    }
+
+    private javax.swing.JPanel getHistoryPanel() {
+        if (panel == null) {
+
+            panel = new javax.swing.JPanel();
+            panel.setLayout(new java.awt.BorderLayout());
+            panel.setName("HTTPSInfo");
+
+            panel.add(this.getTextArea());
+        }
+        return panel;
+    }
+
+    private JTextArea getTextArea() {
+        if (tf == null) {
+            tf =
+                    new JTextArea(
+                            "To start an HTTPS assessment choose 'Scan HTTPS Configuration' from the context menu (right click) on a Sites Tree or History entry");
```

Comment:
This needs to be internationalised, i.e. add the message to the `Messages.properties` file and then get it with `Constant.messages.getString(...)`.

Reply:
Okay I understand this and will make the changes and commit them soon.